custom_components/ui_lovelace_minimalist/configure.py

```diff
@@ -15,6 +15,7 @@
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 LANGUAGES = {
+    "Czech": "CS",
```

Comment:
Please forgive my ignorance. But shouldn't the ISO country code for the Czech Republic be "`CZ`" instead of "`CS`"?

Reply:
Interesting, you're right. 
But now we have a mixture of different logics. Either you should use the ISO code of the country and not the locale one for the time being, or we should change everything over to it.
What do you think about it @stokkie90?

Reply:
According to the home assistant [docs](https://developers.home-assistant.io/docs/translations) And looking at this page. Both are possible? https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry

```
Type: region
Subtag: CZ
Description: Czechia
Description: Czech Republic
Added: 2005-10-16
......
Type: language
Subtag: cs
Description: Czech
Added: 2005-10-16
Suppress-Script: Latn
```
But the language is indeed in this case `CS`